docs/observability/reliability-management-slo/dashboards.md

```diff
@@ -62,86 +67,110 @@ Each dashboard contains the following information:<br/><img src={useBaseUrl('img
 
-## SLO as Log Messages
+:::tip
+Once opened in log search, you can click on **Add to Dashboard** button to add SLO dashboard panels to your own custom dashboards.<br/><img src={useBaseUrl('img/observability/add-to-dashboard.png')} alt="add-to-dashboard" width="200"/>
+:::
 
-Sumo Logic continuously computes data for your SLO behind the scenes. This data, which powers your SLO dashboard, is also made available as log messages that conform to the following schema:
 
-* `Time`: timestamp
-* `sloId`: Id of the SLO, as displayed in the SLO dashboard URL
-* `goodCount`: count of good requests, for request-based, and good windows for windows-based SLOs, based on SLO query definition
-* `totalCount`: count of eligible requests for request-based, and eligible windows for windows-based SLOs, based SLO query definition
-* `sloVersion`: version of SLO definition
+## Custom SLO Dashboards
 
-View the schema by executing the following query:
+### SLO Output Data as Log Messages
+
+Sumo Logic continuously computes data for your SLOs behind the scenes. This data, which powers your SLO dashboards, is also made available as log messages, and can be used to build custom dashboards. You can execute the following query to access your SLO data in logs:
 
 ```sql
-_view=sumologic_slo_output sloId="<your-SLO-ID>"
-| where [subquery: _view=sumologic_slo_output sloId="<your-SLO-ID>"
-| max(sloVersion) as sloVersion | compose sloVersion]
--- (replace with a valid SLO Id)
+_view=sumologic_slo_output
 ```
 
-These log messages will be delayed by one hour, as the system ensures consistency to account for ingest delay of source telemetry.
+It has the following schema:
+
+* `Time`: timestamp
+* `sloId`: Id of the SLO, as displayed in the SLO dashboard URL
+* `goodCount`: count of good requests, for request-based, and good windows for windows-based SLOs, based on SLO query definition
+* `totalCount`: count of eligible requests for request-based, and eligible windows for windows-based SLOs, based SLO query definition
+* `sloVersion`: version of SLO definition. The `sloVersion` is only changed whenever there is a change in semantics of the underlying SLI definition. Therefore, the `sloVersion` is incremented by 1 in case of following modifications only:
+   1. Changing <strong>Source</strong> of the SLO. Example: changing <strong>Query Based</strong> to <strong>Monitor Based</strong>.
+   2. Changing <strong>Evaluation Type</strong>. Example: changing <strong>Request-based</strong> to <strong>Window-based</strong> or changing <strong>Window size</strong> of SLO.
+   3. Any changes to SLO Queries. This includes modifying the queries, changing <strong>Query Type</strong>, changing the <strong>Use values from</strong> and changing the <strong>Success Criteria</strong>.
+   4. Changing <strong>Timezone</strong> of SLO. 
```

Comment:
```suggestion
* `sloVersion`: version of SLO definition. The `sloVersion` is only changed whenever there is a change in semantics of the underlying SLI definition. Therefore, the `sloVersion` is incremented by 1 in case of following modifications only:
   * Changing <strong>Source</strong> of the SLO. Example: changing <strong>Query Based</strong> to <strong>Monitor Based</strong>.
   * Changing <strong>Evaluation Type</strong>. Example: changing <strong>Request-based</strong> to <strong>Window-based</strong> or changing <strong>Window size</strong> of SLO.
   * Any changes to SLO Queries. This includes modifying the queries, changing <strong>Query Type</strong>, changing the <strong>Use values from</strong> and changing the <strong>Success Criteria</strong>.
   * Changing <strong>Timezone</strong> of SLO. 
```